api/notd/token_listing_processor.py

```diff
@@ -97,3 +100,48 @@ async def get_opensea_listings_for_tokens(self, registryAddress: str, tokenIds:
             # NOTE(krishan711): sleep to avoid opensea limits
             await asyncio.sleep(0.1)
         return listings
+
+    async def get_looks_rare_listings_for_collection(self, registryAddress: str) -> List[RetrievedTokenListing]:
+        listings = []
+        queryData = {
+        'isOrderAsk':'true',
+        'collection':registryAddress,
+        'status[]':'VALID',
+        'pagination[first]': 100,
+        'sort':'PRICE_DESC',
+        }
+        flag = True
+        assetListings = []
+        while flag:
```

Comment:
this flag thing is not needed, just use while True

Reply:
So when does it know when to break the loop

Reply:
you have the break on line 120